_data/favicons-database.yml

```diff
@@ -1312,3 +1312,7 @@
   hash: 297a81069094d00a052733d3a0537d18
   name: CrushFTP
   source: CrushFTP
+- algorithm: MD5
+  hash: 1db747255c64a30f9236e9d929e986ca
```

Comment:
hmmmm nmap seems to this this hash is plesk

https://github.com/nmap/nmap/blob/master/nselib/data/favicon-db